### PR TITLE
Update README.md with provisional logo

### PR DESCRIPTION
**Pull request by codeofdiego**
This PR includes a provisional logo to be used in the Belt repo until we design a final one. There's currently a task on Space Station to handle this once there's design time available.

### CODE REVIEW

**Review by smaboshe — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+<div align="center" style="padding: 30px 0px 20px 0px;">
```
Comment (rakeshpetit) [inline README.md:1]: I'm not sure if the `30px` on top looks good. Probably `20px` makes it even between top and bottom.

**Review by rakeshpetit — APPROVED** [review]
Looks good and I left a minor comment about padding.

**Review by stevehanson — COMMENTED** [review]
The brown splatters are... not super palatable to me. What would we think of changing them to some other color?

Before | After
--|--
359940861-95171e10-a4cd-474e-9bb2-c95524f39b6a | belt-blue

Happy to share some other color options if we are open to this. I also would probably vote to remove the splatters altogether. Happy to propose a final design, just wanted to gauge direction first!

**Comment by stevehanson** [comment]
Here's a quick revision with the splatters removed and with a more reddish belt and blue background. I chose those since red and blue are both in thoughtbot's [brand colors](https://thoughtbot.com/playbook/our-company/brand/colors), but I also didn't spend much time to get them exact. What do we think? Feel free to ignore these suggestions if I'm overstepping!

belt

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+  <img src="https://github.com/user-attachments/assets/95171e10-a4cd-474e-9bb2-c95524f39b6a" alt="Logo" width=250>
```
Comment (stevehanson) [inline README.md:2]:
```suggestion
  <img src="https://github.com/user-attachments/assets/4cb9fb46-6c96-4ac0-b7f9-8560e44e11d1" alt="Logo" width=250>
```

We worked together offline on this updated image. This is the image shared [in this comment](https://github.com/thoughtbot/belt/pull/50#pullrequestreview-2254560561).

**Review by stevehanson — APPROVED** [review]
Thanks for moving this forward! A rebase should fix the CI error!

**Comment by codeofdiego** [comment]
Thank you all for the reviews!

**Pull request merged**